integration/docker/Dockerfile-dev

```diff
@@ -0,0 +1,112 @@
+#
+# The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+# (the "License"). You may not use this work except in compliance with the License, which is
+# available at www.apache.org/licenses/LICENSE-2.0
+#
+# This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied, as more fully set forth in the License.
+#
+# See the NOTICE file distributed with this work for information regarding copyright ownership.
+#
+
+# Use CentOS as base image for alluxio-dev image
+FROM centos:7
+
+# Note that downloads for *-SNAPSHOT tarballs are not available
+ARG ALLUXIO_TARBALL=http://downloads.alluxio.io/downloads/files/2.7.0-SNAPSHOT/alluxio-2.7.0-SNAPSHOT-bin.tar.gz
+# (Alert):It's not recommended to set this Argument to true, unless you know exactly what you are doing
+ARG ENABLE_DYNAMIC_USER=false
+
+ADD ${ALLUXIO_TARBALL} /opt/
+# if the tarball was remote, it needs to be untarred
+# use ln -s instead of mv to avoid issues with Centos (see https://github.com/moby/moby/issues/27358)
+RUN cd /opt && \
+    (if ls | grep -q ".tar.gz"; then tar -xzf *.tar.gz && rm *.tar.gz; fi) && \
+    ln -s alluxio-* alluxio
+
+RUN if [ ${ENABLE_DYNAMIC_USER} = "true" ] ; then \
+       chmod -R 777 /opt/* ; \
+    fi
+
+ADD dockerfile-common.sh /
+
+# The following libfuse setup is modified from cheyang/fuse2:ubuntu1604-customize to be applied on centOS
+WORKDIR /
+
+ENV MAX_IDLE_THREADS "64"
+
+RUN \
+    yum update -y && yum upgrade -y && \
+    yum install -y kernel-devel ca-certificates pkgconfig wget udev git && \
+    yum install -y gcc gcc-c++ make cmake gettext-devel libtool autoconf && \
+    yum clean all && \
+    ./dockerfile-common.sh install-libfuse
+
+ENV LD_LIBRARY_PATH "/usr/local/lib:${LD_LIBRARY_PATH}"
+
+ARG ALLUXIO_USERNAME=alluxio
+ARG ALLUXIO_GROUP=alluxio
+ARG ALLUXIO_UID=1000
+ARG ALLUXIO_GID=1000
+ARG ENABLE_DYNAMIC_USER=true
+
+# Add Tini for quick work-around for alluxio helm charts
+# - https://github.com/krallin/tini
```

Comment:
it will be better to comment why Tini is required, if you don't know why, we can @ the target person

Reply:
@LuQQiu I added the PR link which contains why we add tini. Do you think it's sufficient?

Reply:
yeah it's ok